ph5/entry_points.py

```diff
@@ -216,6 +216,10 @@ def __init__(self):
                            'ph5.utilities.fix_3chan_texan:main',
                            'For fixing 3-channel Texan data.',
                            type=EntryPointTypes.ALL),
+                EntryPoint('fix_srm',
```

Comment:
This command does not seem to be found when running the PH5 from the command prompt. Here is the output:
fix_srm --help
-bash: fix_srm: command not found

Reply:
Sorry, this is a new command. You have to update the new command to your PH5 by running `pip install -e .`

Reply:
Thanks for the quick feedback. Testing now.